### ISSUE TITLE
Invalid schema when anyOf & properties are used

### ISSUE THREAD

**Issue by carlosvfernandez** [bcherny/json-schema-to-typescript#314]
Hello, let's see if we can solve this together ... I have this schema:

**AiModel.json**
```
{
  "$id": "id",
  "$schema": "http://json-schema.org/draft-07/schema#",
  "title": "***",
  "description": "description",
  "type": "object",
  "additionalProperties": false,
  "anyOf": [
    {
      "$ref": "./properties/externalConfiguration.json"
    },
    {
      "$ref": "./properties/internalConfiguration.json"
    }
  ],
  "properties": {
    "active": {
      "$ref": "properties/active.json"
    },
    "author": {
      "$ref": "properties/author.json"
    }
}
```

And the result is: 
`export type AIModel = ExternalModel | InternalModel;`

I don't have any object of my properties here...

Thank you in advance.

**Comment by carlosvfernandez** [comment]
Looks like there are like 8 tasks open about this, one is from 2018....

**Comment by bcherny** [comment]
Fixed in 4aabd23, will be published in v10
